app/scripts/views/sign_up.js

```diff
@@ -26,6 +27,7 @@ define(function (require, exports, module) {
   const SignUpMixin = require('views/mixins/signup-mixin');
   const Template = require('stache!templates/sign_up');
 
+  var device = new Device();
```

Comment:
That function `_hasWebChannelSupport` looks like it could live in a mixin, and be used in both views. Here's a suggestion:

1. Create a `user-agent-mixin.js` in https://github.com/mozilla/fxa-content-server/tree/master/app/scripts/views/mixins that includes a `getUserAgent` method in the exported mixin. Bonus if the method caches the first call in something like `this._uap` and then returns that for any future calls.
2. Move the `hasWebChannelSupport` method into the `UserAgent` class in https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/user-agent.js
3. Adjust `devices.js` to use the new `UserAgentMixin`. (So, the view could call `this.getUserAgent()`, and `this.getUserAgent().hasWebChannelSupport()`)
4. Adjust this file (`sign_up.js`) to do the same.